src/helpers/setLocal.js

```diff
@@ -26,5 +26,5 @@ export default (req) => {
 
   const localeData = require('../locale/' + currentLocal + '.js'); // eslint-disable-line
 
-  return localeData.messages;
+  return localeData.default.messages;
```

Comment:
Yes, you need to use `.default` now that we are on webpack 2. We had a plugin that did it automatically and broke tree-shaking of webpack 2.